src/material-experimental/mdc-chips/chip-option.html

```diff
@@ -0,0 +1,28 @@
+<div #mdcChip
+  role="option"
```

Comment:
Could these classes/attributes be set to the element host instead of this div inside the template? The NgClass part is a bit more annoying to do that way, but that's the pattern we try to use across the library.

(similar for the other types of chip below)